src/cclib/parser/utils.py

```diff
@@ -197,4 +193,4 @@ def split(self, line, truncate=True):
 if __name__ == "__main__":
     import doctest
     from cclib.parser import utils
-    doctest.testmod(utils, verbose=False)
+doctest.testmod(utils, verbose=False)
```

Comment:
Changing indents changes the behavior of the code. You've now made `dockets.testmod(utils, verbose=False)` run when this file is imported.

Reply:
Oh yes, Just fixing. Thanks:)

Reply:
Not relevant after merging master in.